apps/server-web/src/main/helpers/services/desktop-server-factory.ts

```
@@ -8,6 +8,7 @@ export class DesktopServerFactory {
 		if (!this.apiInstance && !!env) {
 			this.apiInstance = new WebService(path, env, win, signal, eventEmitter);
 		}
+		this.apiInstance.env = env;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)